src/icon-buttons/icon-button.css

```
@@ -0,0 +1,12 @@
+  .bandicootButton {
+    background-color: transparent;
+    border: none;
+    border-radius: 4px;
+  }
+  .bandicootButton:hover {
+    transition: background-color .25s ease-in-out;
+    background-color: rgba(0, 0, 0, 0.05);
+  }
+  .bandicootButton svg {
+    fill: currentColor;
```

CSS properties are always lowercase. See https://developer.mozilla.org/en-US/docs/Web/CSS/color_value which shows currentcolor all lowercase.

```suggestion
    fill: currentcolor;
```